leaflet/forms/widgets.py

```diff
@@ -39,6 +39,9 @@ def render(self, name, value, attrs=None):
         # Retrieve params from Field init (if any)
         self.geom_type = self.attrs.get('geom_type', self.geom_type)
 
+        # Setting 'loadevent' added in the widget constructor
+        loadevent = self.attrs.get('loadevent', 'load')
```

Comment:
I would suggest to put this in a constant shared with the template tag default value.